compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -29,7 +29,7 @@
 
 logger = logging.getLogger(__name__)
 
-_FUNCX_HOME = os.path.join("~", ".funcx")
+_FUNCX_HOME = os.path.join("~", ".globus_compute")
```

Comment:
".globus_compute" is used in a lot of places, not just source files but also .sh scripts, comments, and documentation.

but if you define it in an SDK file like here (and use it in _FUNCX_HOME), I count 3 places where you can just reference a constant.  That's on the edge of whether it's worth DRY as now there's extra import statements, so I'll leave it up to you.

Reply:
That's a good suggestion, but I think the scope for this PR should be constrained to the re-branding story.